packages/sheets-formula/src/locale/function-list/database/ru-RU.ts

```diff
@@ -0,0 +1,186 @@
+/**
```

Comment:
The translation of the formula part is crawled from the official Excel page, not translated. After we merge it here first, we will update it ourselves. You can pay attention to it in the future.